app/src/test/java/org/groundplatform/android/ui/datacollection/tasks/polygon/DrawAreaTaskViewModelTest.kt

```diff
@@ -171,6 +171,26 @@ class DrawAreaTaskViewModelTest : BaseHiltTest() {
     }
   }
 
```

Comment:
Let's add more tests where the tooltip isn't rendered

Reply:
added